extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/HudiCommands.scala

```diff
@@ -127,6 +127,28 @@ object HudiCommands {
     TableCommandSpec(cmd, Seq(tableDesc), TRUNCATETABLE)
   }
 
+  val CompactionHoodieTableCommand = {
+    val cmd = "org.apache.spark.sql.hudi.command.CompactionHoodieTableCommand"
+    val tableDesc = TableDesc("table", classOf[CatalogTableTableExtractor])
+    TableCommandSpec(cmd, Seq(tableDesc, tableDesc.copy(isInput = true)), CREATETABLE)
+  }
+
+  val CompactionHoodiePathCommand = {
+    val cmd = "org.apache.spark.sql.hudi.command.CompactionHoodiePathCommand"
+    TableCommandSpec(cmd, Seq.empty, CREATETABLE)
```

Comment:
For path-related commands, we just ignore or handle them in another way? Is there any example? cc @yaooqinn @bowenliang123

Reply:
Maybe ignoring path based table first? There's another issue for path-based table #5368. We can align the processing way there.

Reply:
> Maybe ignoring path based table first? There's another issue for path-based table #5368. We can align the processing way there.

Yea, will remain a sub task for compaction path

Reply:
Removed.